ui/ducks/swaps/swaps.js

```diff
@@ -1029,6 +1029,7 @@ export const signAndSendSwapsSmartTransaction = ({
       dispatch(setSwapsSTXSubmitLoading(false));
     } catch (e) {
       console.log('signAndSendSwapsSmartTransaction error', e);
+      dispatch(setSwapsSTXSubmitLoading(false));
```

Comment:
For a future PR, do you think it’d make sense to move `dispatch(setSwapsSTXSubmitLoading(false))` into a finally block since we want it to run either way?